modules/default/weatherforecast/weatherforecast.js

```diff
@@ -141,11 +142,25 @@ Module.register("weatherforecast",{
 
 			var maxTempCell = document.createElement("td");
 			maxTempCell.innerHTML = forecast.maxTemp;
+			if(this.config.scale == "C" || this.config.scale == "c") {
+				maxTempCell.innerHTML += " °C";
+			} else {
+				if(this.config.scale == "F" || this.config.scale == "f") {
+					maxTempCell.innerHTML += " °F";
+				}
+			}
 			maxTempCell.className = "align-right bright max-temp";
 			row.appendChild(maxTempCell);
 
 			var minTempCell = document.createElement("td");
 			minTempCell.innerHTML = forecast.minTemp;
+			if(this.config.scale == "C" || this.config.scale == "c") {
```

Comment:
I consider more elegant to use a lower or upper function case over two conditioanl for a letter.